src/test/java/lotto/LottoNumberFactoryTest.java

```diff
@@ -0,0 +1,33 @@
+package lotto;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LottoNumberFactoryTest {
+
+	@Test
+	@DisplayName("로또번호 6개 생성 확인 테스트")
+	public void createLottoNumberTest() {
+		//given
+		//when
+		List<Integer> numbers = LottoNumberFactory.create();
+		//then
+		assertThat(numbers).hasSize(6);
+	}
+	
+	@Test
+	@DisplayName("로또번호 중복 여부 확인 테스트")
+	public void validateDuplicateLottoNumberTest() {
+		//given
+		List<Integer> numbers = LottoNumberFactory.create();
+		//when
+		List<Integer> distinctNumbers = numbers.stream().distinct().collect(Collectors.toList());
```

Comment:
하나의 라인에 하나의 .을 사용하면 가독성이 좋아집니다 :)

```suggestion
		List<Integer> distinctNumbers = numbers.stream()
                .distinct()
                .collect(Collectors.toList());
```